index.d.ts

```diff
@@ -806,8 +807,7 @@ interface BaseTrailType {
     linkText?: string;
 }
 interface TrailType extends BaseTrailType {
-	design: Design;
-	pillar: Theme;
+    format: Format;
```

Comment:
This is the heart of this PR. All the other changes exist to service this change